src/Hedwig/ClientApp/src/components/ProcessList/_index.scss

```
@@ -0,0 +1,57 @@
+// Derived from VA.gov Design System: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/modules/_m-process-list.scss
+
+.process {
+	list-style: none;
+	padding: 1em 0;
+	position: relative;
+
+	// Exception for when the paragraph is then
+	// lone child. Mostly found in Playbook
+	p {
+		padding: 0;
+		margin-top: 0;
+
+		&:only-of-type {
+			margin-top: 0;
+		}
+	}
+
+	ul {
+		margin: 0 0 1em 1.25em !important;
+		padding: 0;
+
+		li {
+			list-style: square;
+			margin: 0;
+			padding: 0.1em 0;
+		}
+	}
+}
+
+.process-step {
+	border-left: 0.2em solid $theme-color-primary;
+	padding: 0 0 2em 1.55em;
+	list-style: none;
+	margin: 0 !important;
+
+	&:before {
+		color: white;
+		float: left;
+		font-size: 0.8em;
+		font-weight: 700;
+		text-align: center;
+		width: 1.45em;
+		margin-left: -2.8em;
+		display: block;
+		background: white;
+		border: 0.2em solid $theme-color-primary;
+		border-radius: 6em;
+		content: '.';
+		position: relative;
+	}
```

We get rid of `font-size`, because that throws off all the relative sizing (for example, it makes the border 20% narrower).

```suggestion
	&:before {
		display: block;
		content: '';
		float: left;
		width: 1.0625em; /* 17/16 */
		height: 1.0625em; /* 17/16 */
		margin-left: -2.15625em; /* 34.5/16 */
		background: white;
		border: 0.1875em solid $theme-color-primary; /* 3/16 */
		border-radius: 6em;
		position: relative;
	}
```